product-scenarios/1-manage-users-and-roles/1.1-admin-user-manage-roles/1.1.3-manage-roles-with-soap-services/src/test/resources/testng.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Identity-Scenario-Results">
+    <parameter name="useDefaultListeners" value="false"/>
+
+    <listeners>
+        <listener class-name="org.wso2.identity.scenarios.commons.listeners.IdentityScenarioTestListener"/>
+    </listeners>
+
+    <test name="is-groups-mgt-soap-services" preserve-order="true" parallel="false">
+        <classes>
+            <class name="org.wso2.identity.scenarios.test.services.ManagePrimaryRolesWithPermissionTestCase"/>
+            <class name="org.wso2.identity.scenarios.test.services.ManageInternalRolesWithPermissionTestCase"/>
+            <class name="org.wso2.identity.scenarios.test.services.ProvisionUserWithAdminRights"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
any reason to add the blow empty file?

Reply:
test.sh already existing one, no changes done...